pkg/database/mysql/connection.go

```diff
@@ -28,6 +29,12 @@ func (m *MysqlConnection) EngineVersion() string {
 }
 
 func Connect(uri string) (*MysqlConnection, error) {
+	var err error
+	uri, err = ensureMultiStatementsTrue(uri)
```

Comment:
I suggest only supporting this if multistatements are enabled rather than appending this to the connection string.

Reply:
How about instead of multi-statements we add transaction to the ddl?

```
START TRANSACTION;
set @max = (select coalesce(max(`id`), 0) + 1 from `users`);
set @alter_statement = concat('alter table `users` change column `id` `id` int (11) not null auto_increment, auto_increment=', @max);
prepare stmt from @alter_statement; execute stmt; deallocate prepare stmt;
COMMIT;
```

and in code add a hardcoded statement when transaction is detected to 
`defer ROLLBACK`